site/docs/components/overlay/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Overlay
+data:
+  description: "`Overlay` reveals supplementary content when the user clicks a UI trigger element. It remains active and open until the user dismisses it. It can contain interactive and focusable elements, such as buttons and links, in addition to text elements."
```

Comment:
I would scrap `in addition to text elements`. The users can add whatever html content, so I feel it's better to not mention it.